src/context.rs

```diff
@@ -130,10 +137,14 @@ impl Context {
                 WindowEvent::MouseWheel { delta, .. } => match delta {
                     MouseScrollDelta::LineDelta(x, y) => {
                         let line_height = 24.0;
-                        self.raw_input.scroll_delta = Vec2::new(*x, *y) * line_height;
+                        self.raw_input
```

Comment:
accidentally, this change requires egui 0.16. I suppose if it's squashed it does not matter. This commit will not compile.